2018-10-31T11:23:20.331Z - [32minfo[39m: Data Base connection success ...
2018-10-31T11:23:20.381Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 401 but got 200 date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=158564352, heapTotal=123375616, heapUsed=100244808, external=1194834, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=95, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 401 but got 200,     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:95:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.399Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 403 but got 200 date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=162889728, heapTotal=126521344, heapUsed=93412624, external=83570, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=107, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 403 but got 200,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:107:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.409Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=162889728, heapTotal=127569920, heapUsed=94113104, external=92971, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=139, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:139:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.413Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163160064, heapTotal=127569920, heapUsed=94597216, external=94129, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=165, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:165:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.427Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 204 but got 200 date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163160064, heapTotal=127569920, heapUsed=97187664, external=103291, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.end, line=193, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 204 but got 200,     at chai.request.put.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:193:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.434Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 404 but got 200 date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163160064, heapTotal=127569920, heapUsed=99597000, external=103797, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.send.end, line=208, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 404 but got 200,     at chai.request.put.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:208:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.446Z - [31merror[39m: uncaughtException: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163160064, heapTotal=127569920, heapUsed=100133272, external=112997, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=244, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:244:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.448Z - [31merror[39m: uncaughtException: expected { status: false } to equal {} date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163160064, heapTotal=127569920, heapUsed=100463864, external=113479, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=257, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false } to equal {},     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:257:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T11:23:20.455Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 15:23:20 GMT+0400 (+04), pid=27265, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=163430400, heapTotal=127569920, heapUsed=100892416, external=122419, loadavg=[0.77783203125, 1.1083984375, 1.30224609375], uptime=106100, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=274, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:274:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
